.travis.yml

```diff
@@ -2,11 +2,10 @@ language: d
 
 matrix:
   include:
-    - d: dmd
+    # 2.078.1-beta.1 needed for coverage fix (Issue 13742)
+    - d: dmd-beta
       script:
-        # requires fix for Issue 13742
-        - dub test #-b unittest-cov
-        - dub build -b ddox
```

Comment:
Why is this line deleted? is it somehow not necessary anymore to build the docs?

Reply:
My bad, it's still neeeded of course @thaven.